File: 백준/Bronze/1159. 농구 경기/농구 경기.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n;

        n = sc.nextInt();

        sc.nextLine();

        int[] firstChar = new int[26];

        for (int i = 0; i < n; i++) {
            String name;
            name = sc.nextLine();

            firstChar[name.charAt(0) - 'a']++;
        }

        boolean overFive = false;

        for (int i = 0; i < 26; i++) {
            if (firstChar[i] >= 5) {
                overFive = true;
                System.out.printf("%c", (char) (i + 'a'));
            }
        }

        if (overFive == false) {
            System.out.print("PREDAJA");
            ;
        }

        sc.close();
    }
}

File: 백준/Bronze/1159. 농구 경기/농구 경기.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main() {
	int n;

	scanf("%d", &n);

	int firstChar[26] = { 0, };

	for (int i = 0; i < n; i++) {
		char name[30];
		scanf("%s", name);

		firstChar[name[0] - 'a']++;
	}

	int overFive = 0;

	for (int i = 0; i < 26; i++) {
		if (firstChar[i] >= 5) {
			overFive = 1;
			printf("%c", (char)(i + 'a'));
		}
	}

	if (overFive == 0) {
		printf("PREDAJA");
	}

	return 0;
}